src/worker.cc

```diff
@@ -1157,8 +1166,11 @@ void ParseApi(const http_request_t& request, valhalla::Api& api) {
     throw valhalla_exception_t{101};
   };
 
-  // set the action
+  // this is a service request
   api.Clear();
+  api.mutable_info()->set_is_service(true);
```

Comment:
for the http service path we immedately mark the request as being handled by the http service